src/services/utils/index.js

```diff
@@ -40,6 +40,22 @@ export const getLastPrice = (product) => {
   return 0;
 };
 
+/**
+ * Get Avarage Price.
```

Comment:
Small typo with avarage being average.

Following your convention on the function below this one, you could add:

 * @param {Array} prices - Array of prices.
 * @returns {string} - Average price rounded to two decimal places.
 */